src/carve/main.clj

```diff
@@ -44,21 +44,27 @@
   [p]
   (.exists (io/file p)))
 
-(defn validate-opts!
+(defn validate-config!
```

Comment:
No need to rename this function. I think the name is appropriate since it validates the `--opts` parameter.

Reply:
Yes ok cool I renamed it back.
It was a bit weird to have opts being called opts also after they are have been parsed and I liked more `config` as a name, but opts is also fine of course.